### PR TITLE
Watchdog fixes

### PR DESCRIPTION
**Pull request by prymitive**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,11 @@ This option can also be set using `-alertmanager.ttl` flag. Example:
 
 Default is `1m`.
 
+Note that maximum value for this option is `15m`.
```
Comment (jamesog) [inline README.md]: "Note that the maximum"

File: README.md [hunk 2]
```diff
@@ -97,6 +97,11 @@ This option can also be set using `-alertmanager.ttl` flag. Example:
...
+UI have a watchdog feature that will track the timestamp of last pull, if UI
```
Comment (jamesog) [inline README.md]: "The UI has a watchdog that tracks the timestamp of the last pull. If the UI"

File: README.md [hunk 3]
```diff
@@ -97,6 +97,11 @@ This option can also be set using `-alertmanager.ttl` flag. Example:
...
+receiving updates for more than 15 minutes it will print an error and
```
Comment (jamesog) [inline README.md]: Should this be "not receiving updates"? If so, make that sentence "If the UI does not receive updates for more than 15 minutes".

Comment (prymitive) [inline README.md]: yup

**Pull request merged**